table-attributes.md

```diff
@@ -76,49 +77,49 @@ Suppose that in a read-only scenario, you try to reduce the periodic read hotspo
 + Prevent the Regions of a table from merging:
 
     ```sql
-    alter table t attributes[=]'merge_option=deny';
+    ALTER TABLE t ATTRIBUTES[=]'merge_option=deny';
     ```
 
 + Allow merging Regions belonging to a table:
 
     ```sql
-    alter table t attributes[=]'merge_option=allow';
+    ALTER TABLE t ATTRIBUTES[=]'merge_option=allow';
     ```
 
 + Reset attributes of a table:
 
     ```sql
-    alter table t attributes[=]default；
+    ALTER TABLE t ATTRIBUTES[=]default；
```

Comment:
should "default" be in upper cases? @dveeden

Reply:
Yes:

- https://docs.pingcap.com/tidb/stable/keywords#D
- https://github.com/pingcap/tidb/blob/e65f548a40acedd51caa67bed9ee44f51baf852e/parser/parser.y#L113

```suggestion
    ALTER TABLE t ATTRIBUTES[=]DEFAULT；
```